swagger.json

```diff
@@ -959,6 +959,176 @@
                 }
             }
         },
+        "/fleet/vehicles/{vehicle_id}/unassigned_driving_segments": {
```

Comment:
Is unassigned driving segment a concept specific to HOS? Should we put this under an HOS namespace?

Reply:
I think the current namespace is ok